src/test/kotlin/com/igorwojda/integer/countupanddown/challenge.kt

```diff
@@ -4,7 +4,7 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun countUpAndDown(n: Int): List<Int> {
-    TODO("not implemented")
+   return (0..n)+(0 until n).reversed()
```

Comment:
Ranges also have downTo so you don't wasted them creating a range and then reversing it.

Reply:
return (0..n)+(n downTo 0)

Reply:
return (0..n)+(n downTo 0)